thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -553,6 +546,72 @@ private String getDimensions(String dataset, String exploreDimensions) throws Ex
     return reorderedExploreDimensions.toString();
   }
 
+  /**
+   * Show the content of merged anomalies whose start time is located in the given time ranges
+   *
+   * @param functionId id of the anomaly function
+   * @param startTimeIso The start time of the monitoring window
+   * @param endTimeIso The start time of the monitoring window
+   * @param anomalyType type of anomaly: "raw" or "merged"
+   * @param applyAlertFiler can choose apply alert filter when query merged anomaly results
+   * @return list of anomalyIds (Long)
+   */
+  @GET
+  @Path("anomaly-function/{id}/anomalies")
+  public List<Long> getAnomaliesByFunctionId(@PathParam("id") Long functionId,
```

Comment:
I put it here primarily because this class host all endpoints that interact with anomaly-funciton.
This particular one does not fit where in AnomaliesResources, where those endpoints are primarily used by anomalies tab in UI -- the return is a list of metadata of anomaly results.
I do not think a customer facing tab need an endpoints to query anomaly by function Id (we already have the one by function name, this is more intuitive and friendly).

Here I used it primarily for the (onboarding) replay process, we only care about the ids (we also can query raw vs merged right now).

So I think this endpoints belongs here.